library/src/main/kotlin/com/liferay/vulcan/consumer/extensions/String+.kt

```diff
@@ -0,0 +1,8 @@
+package com.liferay.vulcan.consumer.extensions
+
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+
+fun String.asDate(format: SimpleDateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.US)): Date? =
+    try { format.parse(this) } catch (parseException: ParseException) { null }
```

Comment:
I've seen this pattern multiple times in extension methods, transform a function that can throw into one that returns an Optional (nullable, in this case). However I'm open to suggestions :)